BUILD.md

```diff
@@ -4,8 +4,33 @@
 
 Clone the project from GitHub:
 ```sh
-git clone --recursive https://github.com/ArduPilot/ardupilot.git
+apt-get update
+apt-get upgrade
+apt-get install gcc-arm-none-eabi
```

Comment:
You shouldn't use this toolchain but the one we provide. The one on Ubuntu repo is frequently broken